packages/components/inputs/rich-text-input/src/rich-text-input.story.js

```diff
@@ -92,8 +93,12 @@ storiesOf('Components|Inputs', module)
             onChange={onChange}
             value={value}
           />
-          <Text.Headline as="h3">Output</Text.Headline>
-          <pre>{value}</pre>
+          <Text.Headline as="h3">
+            {value === initialValue ? 'Initial HTML-Input:' : 'HTML-Output:'}
```

Comment:
After a frustrating debugging session, I added this, to make clear that the output is actually produced by the editor and not just the initial state.